packages/database/src/migrations/20200930141002-AddLaosSchoolsNumberOfSchoolsSupportedByDevelopmentPartnersDashboardReport-modifies-data.js

```diff
@@ -0,0 +1,239 @@
+'use strict';
+
+import { insertObject, arrayToDbString } from '../utilities/migration';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const DASHBOARD_GROUP_CODES = [
+  'LA_Laos_Schools_Country_Laos_Schools_User',
+  'LA_Laos_Schools_Province_Laos_Schools_User',
+  'LA_Laos_Schools_District_Laos_Schools_User',
+];
+
+const REPORTS = [
+  {
+    id: 'Laos_Schools_Number_Of_Pre_Schools_Supported_Development_Partners',
+    name: 'Number of Pre-primary Schools Supported by Development Partners',
+    schoolType: 'Pre-School',
+  },
+  {
+    id: 'Laos_Schools_Number_Of_Primary_Schools_Supported_Development_Partners',
+    name: 'Number of Primary Schools Supported by Development Partners',
+    schoolType: 'Primary',
+  },
+  {
+    id: 'Laos_Schools_Number_Of_Secondary_Schools_Supported_Development_Partners',
+    name: 'Number of Secondary Schools Supported by Development Partners',
+    schoolType: 'Secondary',
+  },
+];
+
+const BASE_DATA_BUILDER_CONFIG = {
+  operation: {
+    operator: 'COMBINE_BINARY_AS_STRING',
+    dataElementToString: {
+      SchCVD022: {
```

Comment:
I note that you have asked in the card about this, but personally I would expect that 'No support' would be displayed simply if none of the other codes are answered 'Yes'. Let's see what Megan gets back to us with.

Reply:
@Alastair-L ah yes, Megan did get back to me on Slack end of last Friday. She's confirmed that we should use `SchCVD022` for No support

Reply:
Cool. If Megan's happy with it so am I 😄